reconcile/utils/saasherder.py

```diff
@@ -271,27 +271,29 @@ def _get_file_contents_github(repo, path, commit_sha):
         except GithubException as e:
             # slightly copied with love from
             # https://github.com/PyGithub/PyGithub/issues/661
-            errors = e.data['errors']
-            # example errors dict that we are looking for
-            # {
-            #    'message': '<text>',
-            #    'errors': [{
-            #                  'resource': 'Blob',
-            #                  'field': 'data',
-            #                  'code': 'too_large'
-            #               }],
-            #    'documentation_url': '<url>'
-            # }
-            for error in errors:
-                if error['code'] == 'too_large':
-                    # get large files
-                    tree = repo.get_git_tree(
-                        commit_sha, recursive='/' in path).tree
-                    for x in tree:
-                        if x.path != path.lstrip('/'):
-                            continue
-                        blob = repo.get_git_blob(x.sha)
-                        return base64.b64decode(blob.content).decode("utf8")
+            if 'errors' in e.data:
+                errors = e.data['errors']
```

Comment:
```suggestion
            errors = e.data.get('errors', [])
```
let's try to keep this big-ass arrow a little flatter.

Reply:
Apologies if I don't understand well the suggestion, but unfortunately that's not going to make things less nested as there's a chance that we don't return from inside the condition. If I go with your suggestion I would just write:

```python
errors = e.data.get('errors', [])
if errors:
    # (... we may return here or not ...)

raise e
```